src/JustSaying/JustSayingBus.cs

```diff
@@ -40,6 +43,7 @@ public sealed class JustSayingBus : IMessagingBus, IMessagePublisher, IDisposabl
     public JustSayingBus(
         IMessagingConfig config,
         IMessageSerializationRegister serializationRegister,
+        IMessageReceiveController messageReceiveController,
         ILoggerFactory loggerFactory,
         IMessageMonitor monitor)
```

Comment:
This would be a binary-breaking change - to avoid that, we should add a constructor overload and allow this to be null.